compute/cloud-client/instances/src/create_instance.php

```diff
@@ -82,12 +84,12 @@ function create_instance(
     $operation = $instancesClient->insert($instance, $projectId, $zone);
 
     // Wait for the create operation to complete.
```

Comment:
nit: improve the comment
```suggestion
    // Wait for the create operation to complete using a custom helper function.
    // @see src/wait_for_operation.php
```